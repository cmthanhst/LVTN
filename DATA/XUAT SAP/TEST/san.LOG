

 SAP2000 v14.2.2 Advanced (Analysis Build 8851/32)
 File: D:\LVTN\XUAT SAP\TEST\san.LOG




 B E G I N   A N A L Y S I S                                2020/07/18  22:03:12


 RUNNING ANALYSIS WITHIN THE GUI PROCESS
 USING THE ADVANCED SOLVER (PROVIDES LIMITED INSTABILITY INFORMATION)


 E L E M E N T   F O R M A T I O N                                      22:03:12



 L I N E A R   E Q U A T I O N   S O L U T I O N                        22:03:12

 FORMING STIFFNESS AT ZERO (UNSTRESSED) INITIAL CONDITIONS

 TOTAL NUMBER OF EQUILIBRIUM EQUATIONS     =         156
 NUMBER OF NON-ZERO STIFFNESS TERMS        =        3318

 NUMBER OF EIGENVALUES BELOW SHIFT         =           0


 L I N E A R   S T A T I C   C A S E S                                  22:03:12

 USING STIFFNESS AT ZERO (UNSTRESSED) INITIAL CONDITIONS

 TOTAL NUMBER OF CASES TO SOLVE            =           3
 NUMBER OF CASES TO SOLVE PER BLOCK        =           3

 LINEAR STATIC CASES TO BE SOLVED:

 CASE: DEAD
 CASE: TT
 CASE: HT


 A N A L Y S I S   C O M P L E T E                          2020/07/18  22:03:12
