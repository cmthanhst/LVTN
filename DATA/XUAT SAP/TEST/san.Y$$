  RespType: Displ    ,  File: .Y00,  Records:           0        4320
  RespType: React    ,  File: .Y03,  Records:           0        4320
  RespType: JFJoint  ,  File: .Y01,  Records:           0        4320
  RespType: JFSpring ,  File: .Y02,  Records:           0        4320
