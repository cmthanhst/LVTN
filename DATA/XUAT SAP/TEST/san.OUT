 SAP2000 v14.2.2 Advanced (Analysis Build 8851/32)                              
 File: D:\LVTN\XUAT SAP\TEST\san.OUT                                            
                                                                     
                                                                       

 D I S P L A C E M E N T   D E G R E E S   O F   F R E E D O M

         (A) = Active DOF, equilibrium equation
         (-) = Restrained DOF, reaction computed
         (+) = Constrained DOF
         (>) = External substructure DOF
         ( ) = Null DOF

  JOINTS              UX  UY  UZ  RX  RY  RZ
       2 TO     448    -   -   -   -   -   -
     449 TO     474    A   A   A   A   A   A
